### ISSUE TITLE
[vcpkg upgrade] Could not find rawparse in gstreamer.

### ISSUE THREAD

**Issue by UnknownObject000** [microsoft/vcpkg#29750]
**Describe the bug**
I tried to upgrade my librarys using ```vcpkg upgrade```, but it pops an error like that:
```
Could not find rawparse in gstreamer.
note: updating vcpkg by rerunning bootstrap-vcpkg may resolve this failure.
```

**Environment**
- OS: Windows 11 22H2 Profrssional Workstation
- Compiler: Visual Studio 2022 (MSVC17)

**To Reproduce**
```
PS F:\vcpkg> git pull
Already up to date.
PS F:\vcpkg> vcpkg upgrade
Could not find rawparse in gstreamer.
note: updating vcpkg by rerunning bootstrap-vcpkg may resolve this failure.
PS F:\vcpkg> .\bootstrap-vcpkg.bat
Downloading https://github.com/microsoft/vcpkg-tool/releases/download/2023-01-24/vcpkg.exe -> F:\vcpkg\vcpkg.exe (using proxy: http://127.0.0.1:30100)... done.
Validating signature... done.

Telemetry
---------
vcpkg collects usage data in order to help us improve your experience.
The data collected by Microsoft is anonymous.
You can opt-out of telemetry by re-running the bootstrap-vcpkg script with -disableMetrics,
passing --disable-metrics to vcpkg on the command line,
or by setting the VCPKG_DISABLE_METRICS environment variable.

Read more about vcpkg telemetry at docs/about/privacy.md
PS F:\vcpkg> .\vcpkg upgrade
Could not find rawparse in gstreamer.
note: updating vcpkg by rerunning bootstrap-vcpkg may resolve this failure.
```

**Expected behavior**
Packages be upgraded.

**Failure logs**
```
Could not find rawparse in gstreamer.
note: updating vcpkg by rerunning bootstrap-vcpkg may resolve this failure.
```

**Comment by autoantwort** [comment]
Iiuc solved by https://github.com/microsoft/vcpkg/pull/29664

**Comment by UnknownObject000** [comment]
> Iiuc solved by #29664

Downloaded Pre-Release 2023-02-16, fixed.
